.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 install: npm install
 script:
   - npm test
-  - nsp check
+  - nsp check; exit 0
```

Comment:
From now on Snyk will handle NSP checks on out PR's. So we can tell travis not the fail our builds if vulnerabilities were not added in out PR's, but existed beforehand.